python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py

```diff
@@ -426,7 +426,10 @@ def insert_data_tables(
         raise error
 
 
-def process_static_tables(db_manager: DatabaseManager) -> None:
+def process_static_tables(
+    rpm_db_manager: DatabaseManager,
+    md_db_manager: DatabaseManager,
+) -> None:
```

Comment:
we need the md db manager to figure out what the new feed info files are and the rpm db manager to load the new data into.